### PR TITLE
Ignore coverage path

### PR DESCRIPTION
**Pull request by kevin940726**
1. ignore lib folder in coverage path
2. ignore coverage folder in git

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,10 @@
     "jest": "^20.0.0"
   },
   "jest": {
-    "collectCoverage": true
+    "collectCoverage": true,
+    "coveragePathIgnorePatterns": [
+      "<rootDir>/lib/",
+      "<rootDir>/node_modules/"
```
Comment (boczeratul) [inline package.json]: I thought `node_modules` is ignored by default?

**Comment by boczeratul** [comment]
can you also help adding coverage to `.npmignore` ?

File: package.json (hunk 1 above)
Comment (kevin940726) [inline package.json]: Yes, but since we override it here I think we have to explicitly add it?

Comment (boczeratul) [inline package.json]: you're right 👍

**Pull request merged**